### PR TITLE
Docs for torch.*_like(...) factory functions

### PR DESCRIPTION
**Pull request by zou3519**
In the same spirit as `torch.randn_like`.

cc @gchanan @SsnL @li-roy

### CODE REVIEW

**Comment by zou3519** [comment]
`zeros_like` and `ones_like` used to support an `out=` keyword but no longer do. Should this be a cause for concern?

**Comment by SsnL** [comment]
The original `out=` should be equivalent with `.resize(*).fill_(1)`. Maybe we should add a note about it? I doubt many people use it though, so a deprecated version in PyThon is probably not needed?

**Comment by gchanan** [comment]
I'd be surprised if it's used; can you add that as a release note?

**Comment by zou3519** [comment]
Will do

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -5749,14 +5762,27 @@
...
 
 Returns a tensor filled with the scalar value `0`, with the same size as
-:attr:`input`.
+:attr:`input`. ``torch.zeros_like(input)`` is equivalent to
+``torch.zeros(input.size(), dtype=input.dtype, layout=input.layout, device=input.device)``.
+
+.. warning::
+    As of 0.4, this function does not support an :attr:`out` keyword. As an alternative,
+    the old ``torch.zeros_like(input, out=output)`` is equivalent to
+    ``output.resize_as_(input).fill_(0.)``.
```
Comment (gchanan) [inline torch/_torch_docs.py]: doesn't `torch.ones` still have an `out`?  Can we use that instead?

**Review by gchanan — APPROVED** [review]

**Pull request merged**